lib/spark/elixir_sense/entity.ex

```diff
@@ -27,7 +29,9 @@ defmodule Spark.ElixirSense.Entity do
       if builtins && !String.contains?(hint, ".") && lowercase_string?(hint) do
         apply(Complete, :complete, [
           to_string("#{inspect(builtins)}.#{hint}"),
```

Comment:
we should likely first check to see if we are on the old version or the new version using something like `function_exported?(Complete, :complete, 4)` that way this supports both versions.

Reply:
you're right!